content/apps/heatmap/contents.lr

```diff
@@ -0,0 +1,21 @@
+_model: app
+---
+title: heatmap
+---
+author: Folkert van Heusden
+---
+category: website
+---
+description: This website collects the openclosed status of all hackerspaces known by spaceapi.io and shows for each a heatmap of their openclosed times.
```

Comment:
Hm, I'm a bit confused. On the website itself you write something about a space being "always reachable". I assumed this is about the SpaceAPI endpoint being reachable, right?

If it's about the opening status of the space, then something must be wrong, since at least Coredump is closed right now and should not be green.

Reply:
If you click on it you see the open closed stats. The color on the overview page only shows reach-ability of the API endpoint.

Reply:
Ha, I never noticed that feature :smile: Cool!